### PR TITLE
Updated german translations (de-DE)

### PR DESCRIPTION
**Pull request by gannebamm**
## Description
The de-DE.json was outdated and needed some translation work. As discussed with @allyoucanmap here https://github.com/GeoNode/geonode-mapstore-client/pull/760#pullrequestreview-865154214

A new PR against this repository was created

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: translations

## Issue

**What is the current behaviour?**
Some strings had no or bad translations

**What is the new behaviour?**
The overall quality of the german translation was improved

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by gannebamm** [comment]
If there is something left to do for this PR please ping me.

File: web/client/translations/data.de-DE.json [hunk 1]
```diff
@@ -654,11 +654,7 @@
                 "interval_limit": "Intervalle sollten eine Zahl zwischen {min} und {max} sein",
                 "invalid_object": "Ungültige Serviceantwort",
                 "invalid_geometry": "Der Geometrietyp ist nicht gültig, kein Punkt, Linie oder Polygon",
-                "invalid_attribute": "Ungültiger Attributtyp bei Auswahl. Bitte wählen Sie ein gültiges Attribut aus der Liste für die Klassifizierungsmethode aus",
-                "invalid_classes": "Max muss in jeder Klasse größer als min sein"
-            },
-            "error": {
-                "defaultGroupOnTop": "Die Gruppe {groupTitle} ist als Standardzielgruppe festgelegt und muss daher oben bleiben"
+                "invalid_classes": "Maximum muss in jeder Klasse größer als Minimum sein"
             }
```
Comment (allyoucanmap) [inline web/client/translations/data.de-DE.json:658]:
```suggestion
                "invalid_attribute": "Ungültiger Attributtyp bei Auswahl. Bitte wählen Sie ein gültiges Attribut aus der Liste für die Klassifizierungsmethode aus",
                "invalid_classes": "Maximum muss in jeder Klasse größer als Minimum sein"
            },
            "error": {
                "defaultGroupOnTop": "Die Gruppe {groupTitle} ist als Standardzielgruppe festgelegt und muss daher oben bleiben"
            }
```

**Comment by gannebamm** [comment]
Sorry @allyoucanmap I was ill. I will work on this tomorrow

**Comment by allyoucanmap** [comment]
@gannebamm don't worry the last commit should solve the error in the testing checks, if everything passes I'm going to merge, thanks again for the contribution

**Review by allyoucanmap — APPROVED** [review]

**Comment by allyoucanmap** [comment]
@ElenaGallo kindly test this update in DEV and let me know when I can backport on the 2022.02.xx branch

**Pull request merged**